plugins/generic/customBlockManager/CustomBlockEditForm.inc.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @file CustomBlockEditForm.inc.php
+ *
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @package plugins.generic.customBlockManager
+ * @class CustomBlockEditForm
+ *
+ * Form for site admins to create and modify sidebar blocks
+ *
+ */
+
+import('lib.pkp.classes.form.Form');
+
+class CustomBlockEditForm extends Form {
+	/** @var $plugin object */
+	var $plugin;
+
+	/** $var $errors string */
+	var $errors;
+
+	/**
+	 * Constructor
+	 */
+	function CustomBlockEditForm(&$plugin) {
+
+		parent::Form($plugin->getTemplatePath() . 'editCustomBlockForm.tpl');
+
+		$this->plugin =& $plugin;
+
+		$this->addCheck(new FormValidatorPost($this));
+		$this->addCheck(new FormValidator($this, 'blockContent', 'required', 'plugins.generic.customBlock.contentRequired'));
+
+	}
+
+	/**
+	 * Initialize form data from current group group.
+	 */
+	function initData() {
+		$plugin =& $this->plugin;
+
+		// add the tiny MCE script 
+		$this->addTinyMCE();
```

Comment:
We do, but this plugin and the static pages plugin provide their own customized instance.